lib/instana/activators/sequel.rb

```diff
@@ -0,0 +1,21 @@
+# (c) Copyright IBM Corp. 2021
+# (c) Copyright Instana Inc. 2021
+
+module Instana
+  module Activators
+    class Sequel < Activator
+      def can_instrument?
+        defined?(::Sequel::Database)
```

Comment:
Ideally, tha added option in `config.rb`, would be used here already. In fact I don't see anywhere in production code  that you read `config[:sequel]`.